api/app/db/models/weather_models.py

```diff
@@ -183,6 +183,8 @@ class ModelRunPrediction(Base):
         "PredictionModelRunTimestamp", foreign_keys=[prediction_model_run_timestamp_id])
     # The date and time to which the prediction applies.
     prediction_timestamp = Column(TZTimeStamp, nullable=False, index=True)
+    # The station code representing the location (aka weather station).
+    station_code = Column(Integer, nullable=True)
```

Comment:
Is it possible for the station code to be null?

Reply:
Good question....if we keep the old data around then yes, the `station_code` field for old records will be null. Now that I think about it we don't need to keep the old data around as it isn't helpful without the `station_code` field (ie. it can't be used for bias adjustments). I can add a line to the migration to delete all the records from the table and make `station_code` non-null. What do you think?

Reply:
Oh good points. A third option is to delete old records and make `station_code` non-null in a follow up migration. Up to you how you'd like to do it, but I think getting the table to the state where all records have a non-null `station_code` is ideal.

Reply:
I tweaked the existing migration to drop the records before adding the field and tested locally.